packages/mds-stream/stream-interface.ts

```diff
@@ -0,0 +1,8 @@
+import { VehicleEvent, Telemetry, Device } from '@mds-core/mds-types'
+
+export interface Stream {
+  writeEvent: (event: VehicleEvent) => Promise<void>
+  writeTelemetry: (telemetry: Telemetry[]) => Promise<void>
+  writeDevice: (device: Device) => Promise<void>
+  shutdown: () => void
```

Comment:
I would add `initialize` for symmetry even if it's just a no-op for now? Also, would probably make `shutdown` async also.

Reply:
I could write in `initialize` and use it... But I can't seem to think of a clean way to do it, and _really_ hate having to use `let` for it.